### PR TITLE
B refactor postpone

### PR DESCRIPTION
**Pull request by TanYiXiang**
Refactor postpone command to be more human readable.

### CODE REVIEW

File: src/main/java/chronologer/command/ExitCommand.java [hunk 1]
```diff
@@ -1,30 +1,24 @@
...
 
     /**
      * Terminates the program by updating isExit to true.
      */
-    public void execute(TaskList tasks, Storage storage) throws ChronologerException {
-        super.commandOut();
-        UiTemporary.printGoodbye();
-        UiTemporary.printOutput(GOODBYE);
+    public void execute(TaskList tasks, Storage storage)  {
+        System.exit(0);
```
Comment (fauzt) [inline src/main/java/chronologer/command/ExitCommand.java]: maybe reduntant, but should we retain the UiTemporary printout and have the System.exit(0) line appended after it?

Comment (TanYiXiang) [reply src/main/java/chronologer/command/ExitCommand.java]: It won't show up, I tried. The system exits faster than it prints. You can pause using a thread function but then it throws a different exception which will affect the parent command class so I decided to left it out for now.

Comment (fauzt) [reply src/main/java/chronologer/command/ExitCommand.java]: ok we'll try figure something out if we need to retain the exit message later on

Comment (E0310898) [reply src/main/java/chronologer/command/ExitCommand.java]: Ah dont worry i have a quick save feature done up! should be fine!

**Pull request merged**